aiohttp/client.py

```diff
@@ -369,11 +363,7 @@ async def _request(
         if self.closed:
             raise RuntimeError("Session is closed")
 
-        if not isinstance(ssl, SSL_ALLOWED_TYPES):
-            raise TypeError(
-                "ssl should be SSLContext, bool, Fingerprint, "
-                "or None, got {!r} instead.".format(ssl)
-            )
```

Comment:
This can probably just be reverted, as the check remains simple now.

Reply:
This is moved into method verify_ssl_type()

Reply:
Yes, I'm just not sure we need a method anymore given it is a very simple check. It would make the review easier if we can minimise unrelated changes here.